api/v1beta2/foundationdb_version.go

```diff
@@ -224,4 +230,5 @@ var Versions = struct {
 	SupportsRocksDBV1:      Version{Major: 7, Minor: 1, Patch: 0, ReleaseCandidate: 4},
 	SupportsIsPresent:      Version{Major: 7, Minor: 1, Patch: 4},
 	SupportsShardedRocksDB: Version{Major: 7, Minor: 2, Patch: 0},
+	SupportsRecoveryState:  Version{Major: 7, Minor: 1, Patch: 22},
```

Comment:
It exists since 7.0.0 but it's broken since then: https://github.com/apple/foundationdb/pull/8067 that's the reason I added the `7.1.22` version here. That is the version that will contain the fix (https://github.com/apple/foundationdb/pull/8068)